_posts/2025-02-19-arrow-java-18.2.0.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title: "Apache Arrow Java 18.2.0 Release"
+date: "2025-02-19 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+The Apache Arrow team is pleased to announce the [v18.2.0](https://github.com/apache/arrow-java/releases/tag/v18.2.0) release of Apache Arrow Java. 
+This is the first release since Arrow Java landed in its own [repository](https://github.com/apache/arrow-java).
+
+## Changelog
+
+### What's Changed
```

Comment:
Is this changelog generated by Github?

Reply:
I think so.

We may not need this in this release post because we can see this in https://github.com/apache/arrow-java/releases/tag/v18.2.0 and this page is linked.

Can we pick up only feature improvements and bug fixes in this list? (Can we remove apache/arrow-java setup related changes?)

Reply:
Yes, agree. As the diff is not available on arrow-java (18.1.0 was not from this repo), I wasn't sure what we wanted to include.